### PR TITLE
React: Calculator app.

### PR DESCRIPTION
**Pull request by FrederickMih**
## :arrow_forward: Calculator: Testing phase.
    
       In this Milestone, 

:diamond_shape_with_a_dot_inside:  I wrote Snapshot Test for the pages below:
- Home,
- Calculator, and
- Quote

:diamond_shape_with_a_dot_inside:  I equally wrote test suites for the logic functions below:
- `calculate()` and, 
- `operate()`

### CODE REVIEW

File: src/styles/Navbar.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+/* @keyframes menu-logo {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+} */
```
Comment (bellom) [inline src/styles/Navbar.css]: Kindly remove commented code

File: src/styles/Navbar.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  justify-content: space-around;
+  width: 15rem;
+  align-items: center;
+  font-size: 1.2rem;
+}
+
+/* .menu-logo {
+  height: 5vmin;
+  pointer-events: none;
+} */
```
Comment (bellom) [inline src/styles/Navbar.css]: It’s better to delete commented code. If you need to review a previous version of your code, use git history. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

File: src/tests/calculate.test.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    ).not.toEqual('3');
+  });
+
+  it('should subtract next number from the total', () => {
+    expect(
+      calculate({ total: '5', next: '1', operation: '-' }, '-').total,
+    ).toEqual('4');
+  });
+
+  it('should compute incorect subtraction', () => {
```
Comment (bellom) [inline src/tests/calculate.test.js]: Let's make sure our words are readable, `incorrect` instead of `incorect`

File: src/tests/calculate.test.js [hunk 4]
```diff
@@ -0,0 +1,94 @@
+import calculate from '../logic/calulate';
+
+describe('Testing arithmetic operations', () => {
+  it('sum up', () => {
```
Comment (bellom) [inline src/tests/calculate.test.js]: Let's make sure our `it` block sentence is explanatory. `sum up` or `falsy sum` doesn't really explain the error if the test failed.

File: src/tests/calculate.test.js [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+    expect(
+      calculate({ total: '77', next: null, operation: null }, '%').total,
+    ).toEqual(0.77);
+  });
+
+  it("test '.' operator", () => {
+    expect(
+      calculate({ total: '5', next: '3', operation: null }, '.').total,
+    ).not.toEqual('5.3');
+  });
```
Comment (bellom) [inline src/tests/calculate.test.js:104]: Let's add one more test for the following condition in `calculate.js`

```
  if (btnName === '.') {
    if (next && !next.includes('.')) {
      next += '.';
    } else if (total && !total.includes('.')) {
      total += '.';
    }
  }
```

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @FrederickMih  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check my comments under this review;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best**

**Review by Elbie-em — APPROVED** [review]
## STATUS:  APPROVED ✅

Hi @FrederickMih, great job implementing all changes requested by the previous TSE, your project is complete!  There is nothing else to say other than... it's time to merge it.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

____________

> Elbie Moonga (Part-time TSE)

> Github (@Elbie-em)
> Slack (Elbie Moonga)
> Twitter ([@ElbieEm](https://twitter.com/ElbieEm))
> LinkedIn([Elbie Moonga](https://www.linkedin.com/in/elbiemoonga/))

**Pull request merged**